package-lock.json

```diff
@@ -0,0 +1,1442 @@
+{
+  "name": "crashpad",
+  "version": "1.3.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "@ungap/promise-all-settled": {
+      "version": "1.1.2",
+      "resolved": "https://npm.goodeggs.com/@ungap%2fpromise-all-settled/-/promise-all-settled-1.1.2.tgz",
```

Comment:
ugh, this repo is misconfigured... before you run `yarn`, can you please fix this? https://goodeggs.atlassian.net/wiki/spaces/ENG/pages/459833453/Using+NPM+and+Yarn#How-to-configure-a-public-module. It's missing an `.npmrc` that says to use the public registry and our dev machines are configured to use our private registry by default. See https://goodeggs.atlassian.net/wiki/spaces/ENG/pages/459833453/Using+NPM+and+Yarn#I-ran-yarn-add-and-now-there-are-references-to-npm.goodeggs.com-in-yarn.lock-in-a-public-module!.